charts/prometheus-redis-exporter/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v1.56.0
+appVersion: v1.57.0
 description: Prometheus exporter for Redis metrics
 name: prometheus-redis-exporter
-version: 6.1.0
+version: 6.2.0
```

I feel bumping to patch version should be fine.

```suggestion
version: 6.1.1
```